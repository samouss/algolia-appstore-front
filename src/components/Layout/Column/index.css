.Column {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  flex-wrap: wrap;
  width: 100%;
  margin-top: 1vh;
  // background-color: blue;
  // border: 1px solid #f2f2f2;

  &:first-child {
    margin-top: 0;
    margin-left: 0;
  }

  @media (--tablet) {
    width: 25%;
    margin-top: 0;
    margin-left: 1vh;
  }
}
