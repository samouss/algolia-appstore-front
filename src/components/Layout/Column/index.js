import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const Column = ({ children }) => (
  <div styleName="Column">
    {children}
  </div>
);

Column.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Column;
