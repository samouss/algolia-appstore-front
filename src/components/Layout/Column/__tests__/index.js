import React from 'react';
import { shallow } from 'enzyme';
import Column from '../index';

describe('<Column />', () => {
  const defaultProps = {};

  it('expect to render', () => {
    const props = {
      ...defaultProps,
    };

    const component = shallow(
      <Column
        {...props}
      >
        Content
      </Column>,
    );

    expect(component).toMatchSnapshot();
  });
});
