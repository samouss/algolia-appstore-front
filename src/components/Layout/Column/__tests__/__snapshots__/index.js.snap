// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Column /> expect to render 1`] = `
<div
  styleName="Column"
>
  Content
</div>
`;
